src/collections/_documentation/api/projects/get-project-details.md

```diff
@@ -1,12 +1,12 @@
true, \n      \"id\": \"vsts\", \n      \"isHidden\": true, \n      \"isTestable\": false, \n      \"metadata\": {}, \n      \"name\": \"Visual Studio Team Services\", \n      \"resourceLinks\": [\n        {\n          \"title\": \"Report Issue\", \n          \"url\": \"https://github.com/getsentry/sentry/issues\"\n        }, \n        {\n          \"title\": \"View Source\", \n          \"url\": \"https://github.com/getsentry/sentry/tree/master/src/sentry_plugins\"\n        }\n      ], \n      \"shortName\": \"VSTS\", \n      \"slug\": \"vsts\", \n      \"status\": \"unknown\", \n      \"type\": \"issue-tracking\", \n      \"version\": \"10.1.0.dev0\"\n    }, \n    {\n      \"assets\": [], \n      \"author\": {\n        \"name\": \"Sentry Team\", \n        \"url\": \"https://github.com/getsentry/sentry\"\n      }, \n      \"canDisable\": true, \n      \"contexts\": [], \n      \"description\": \"\\nTrigger outgoing HTTP POST requests from Sentry.\\n\\nNote: To configure webhooks over multiple projects, we recommend setting up an\\n[Internal Integration](https://docs.sentry.io/workflow/integrations/integration-platform/#internal-integrations).\\n\", \n      \"doc\": \"\", \n      \"enabled\": false, \n      \"featureDescriptions\": [\n        {\n          \"description\": \"Configure rule based outgoing HTTP POST requests from Sentry.\", \n          \"featureGate\": \"alert-rule\"\n        }\n      ], \n      \"features\": [\n        \"alert-rule\"\n      ], \n      \"hasConfiguration\": true, \n      \"id\": \"webhooks\", \n      \"isHidden\": false, \n      \"isTestable\": true, \n      \"metadata\": {}, \n      \"name\": \"WebHooks\", \n      \"resourceLinks\": [\n        {\n          \"title\": \"Report Issue\", \n          \"url\": \"https://github.com/getsentry/sentry/issues\"\n        }, \n        {\n          \"title\": \"View Source\", \n          \"url\": \"https://github.com/getsentry/sentry/tree/master/src/sentry/plugins/sentry_webhooks\"\n        }\n      ], \n      \"shortName\": \"WebHooks\", \n      \"slug\": \"webhooks\", \n      \"status\": \"unknown\", \n      \"type\": \"notification\", \n      \"version\": \"10.1.0.dev0\"\n    }\n  ], \n  \"processingIssues\": 0, \n  \"relayPiiConfig\": null, \n  \"resolveAge\": 0, \n  \"safeFields\": [], \n  \"scrapeJavaScript\": true, \n  \"scrubIPAddresses\": false, \n  \"securityToken\": \"3c7365d26c5111eabb500242ac110002\", \n  \"securityTokenHeader\": null, \n  \"sensitiveFields\": [], \n  \"slug\": \"pump-station\", \n  \"status\": \"active\", \n  \"storeCrashReports\": 0, \n  \"subjectPrefix\": \"[Sentry] \", \n  \"subjectTemplate\": \"$shortID - $title\", \n  \"symbolSources\": null, \n  \"team\": {\n    \"id\": \"2\", \n    \"name\": \"Powerful Abolitionist\", \n    \"slug\": \"powerful-abolitionist\"\n  }, \n  \"teams\": [\n    {\n      \"id\": \"2\", \n      \"name\": \"Powerful Abolitionist\", \n      \"slug\": \"powerful-abolitionist\"\n    }\n  ], \n  \"verifySSL\": false\n}", 
```

Comment:
Why is this so much bigger?

Reply:
The last time this was updated was in 2018. Since then, we have added a lot more plugins and that's what's making this diff larger.